### PR TITLE
[OSDOCS 7424] Restructure IBM Power config parms to book level

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-7424

Link to docs preview:

- [Installation configuration parameters for IBM Power](https://63517--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installation-config-parameters-ibm-power.html)
New assembly at book level
- Installing a cluster on IBM Power [Manually creating the installation configuration file](https://63517--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html#installation-initializing-manual_installing-ibm-power)
Note the absence of Installation configuration parameters following this topic. In liue of the module, a cross reference is provided for the new assembly.
It is worth noting that the latter is an example of the change that this PR introduces. This change has been applied to all IBM Power installation assemblies.

QE review:
- [NA ] QE has approved this change.
  Given that this update is a reorg only, QE approval is not required. See conversation here: https://github.com/openshift/openshift-docs/pull/61165#issuecomment-1600861041. 

Additional information: 
- Related PR https://github.com/openshift/openshift-docs/pull/61165

### CODE REVIEW

**Comment by SNiemann15** [comment]
/label peer-review-needed

@sjstout PTAL

File: installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-ibm-power"]
+= Installation configuration parameters for {ibmpowerProductName}
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-ibm-power
+:platform: IBM Power
+
+toc::[]
+
+Before you deploy an {product-title} cluster, you provide a customized `install-config.yaml` installation configuration file that describes the details for your environment.
```
Comment (skrthomas) [inline installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc:10]:
```suggestion
Before you deploy an {product-title} cluster, provide a customized `install-config.yaml` installation configuration file that describes the details for your environment.
```

Comment (skrthomas) [reply installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc:10]: I think you can omit this 2nd "you".

**Review by skrthomas — COMMENTED** [review]
The reorg looks good to me. I left an unrelated comment for your consideration.

File: installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc (hunk 1 above)
Comment (SNiemann15) [reply installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc:10]: For consistency with the other providers I would prefer keeping the "you". WDYT @mjpytlak ?

Comment (mjpytlak) [reply installing/installing_ibm_power/installation-config-parameters-ibm-power.adoc:10]: > For consistency with the other providers I would prefer keeping the "you". WDYT

Yes @SNiemann15 please keep the second "you".

**Comment by mjpytlak** [comment]
@SNiemann15 This LGTM. When you are ready, please request a merge review and I will take care of this for you.

**Comment by sjstout** [comment]
LGTM :)

**Pull request merged**